src/test/java/com/checkmarx/flow/cucumber/integration/pullrequest/updatecomments/UpdatePullRequestCommentsSteps.java

```diff
@@ -85,7 +96,13 @@ public UpdatePullRequestCommentsSteps(GitHubService gitHubService, GitHubPropert
         this.flowProperties = flowProperties;
         this.cxProperties = cxProperties;
         this.scaProperties = scaProperties;
+        this.authClient = authClient;
+        sastFile = toFullResourcePath(filePath);
```

Comment:
is this a "SAST only" thing?

Reply:
why does the constructor creates this path, if it is from a const?

Reply:
fixed